agenta-web/src/lib/helpers/utils.ts

```diff
@@ -6,7 +6,8 @@ import promiseRetry from "promise-retry"
 import {getErrorMessage} from "./errorHandler"
 import dayjs from "dayjs"
 import utc from "dayjs/plugin/utc"
-dayjs.extend(utc)
+// causing all cypress test to fail
+// dayjs.extend(utc)
```

Comment:
@MohammedMaaz do you have an idea why this is breaking the tests?

Reply:
Can you try adding this line? `Cypress.dayjs = dayjs`